client/reader/tag-stream/main.jsx

```diff
@@ -109,6 +109,23 @@ class TagStream extends React.Component {
 			} );
 		}
 
+		if ( ! tag || tag.error ) {
```

Comment:
the `! tag` bit here is causing the fragment below to render while we're loading the tag info. I think we may only want to check for `tag.error` ?

Reply:
good idea